map/src/map/layers/CustomTileLayer.js

```diff
@@ -281,9 +306,16 @@ export function CustomTileLayer({ ...props }) {
             }
         });
 
+        const onMapClick = () => {
```

Comment:
Please use useCallback for map event handlers to avoid multiple on() / incorrect off()